src/components/TransactionHistory/AssetTransactionHistory.tsx

```diff
@@ -29,21 +29,22 @@ export const AssetTransactionHistory: React.FC<AssetTransactionHistoryProps> = (
 }) => {
   const translate = useTranslate()
   const location = useLocation()
-  const generatedPath = `${location.pathname}/transactions`
-  const {
-    state: { wallet },
-  } = useWallet()
-
+  const wallet = useWallet().state.wallet
   const asset = useAppSelector(state => selectAssetById(state, assetId ?? ''))
-  const chainId: ChainId = asset?.chainId ?? ''
 
-  const filter = useMemo(() => ({ assetId, accountId }), [assetId, accountId])
+  const chainId: ChainId = (() => {
```

Comment:
get `chainId` from either `asset` or `accountId` to support account views with no `assetId` passed